apps/editor/src/__test__/unit/eventEmitter.spec.ts

```diff
@@ -201,4 +201,44 @@ describe('eventEmitter', () => {
       expect(handlerBeRemoved).not.toHaveBeenCalled();
     });
   });
+
+  describe('hold event', () => {
+    let handler: jest.Mock;
+
+    function triggerEvent(apiName: 'emit' | 'emitReduce') {
+      if (apiName === 'emit') {
+        emitter.emit('myEvent');
+      } else {
+        emitter.emitReduce('myEvent', 0);
+      }
+    }
+
+    beforeEach(() => {
+      handler = jest.fn();
+
+      emitter.addEventType('myEvent');
+
+      emitter.listen('myEvent', handler);
+    });
+
+    (['emit', 'emitReduce'] as const).forEach((apiName) => {
+      it(`should not call the holding event with ${apiName} API`, () => {
+        emitter.holdEventInvoke(() => {
+          triggerEvent(apiName);
+        });
```

Comment:
```suggestion
emitter.holdEventInvoke(() => triggerEvent(apiName));
```
로 해도 될 것 같습니다. 아래도요!